src/components/ui.js

```diff
@@ -42,6 +42,41 @@ export const HeaderChangeLocaleWrapper = styled(Box)`
   }
 `;
 
+// ///////////////////////////////////////////////////////////
+// Vidoe section
+export const OurDemo = styled(Flex)`
+ display: flex;
+ color: #FBFBFB;
+ font-size: 48px;
+ padding-top: 37px;
+ text-align: center;
+ align-items: center;
+ flex-direction: column;
+ background-color: #363135;
+
+ ${createMaxWithMediaQuery(GC.FIRST_UI_BREAKPOINT)} {
+    font-size: 28px;
+  }
+
+ ${createMaxWithMediaQuery(GC.SECOND_UI_BREAKPOINT)} {
+    font-size: 18px;
+  }
+
+`;
+
+export const UnderLine = styled.div`
```

Comment:
add comment `//TODO: Delete when Title component will  be ready`

Reply:
+